tests/templates/test_embeddings.py

```diff
@@ -59,7 +59,7 @@ def test_amplitude_embedding_prepares_state(self, inpt):
 
         @qml.qnode(dev)
         def circuit(x=None):
-            AmplitudeEmbedding(features=x, wires=range(n_qubits), pad=None, normalize=False)
+            AmplitudeEmbedding(features=x, wires=range(n_qubits), pad=None)
```

Comment:
But now that `normalize=True` by default, isn't this changing the nature of the test?

Reply:
Quite frankly, using `math.isclose` makes using `AmplitudeEmbedding` pretty sensitive in terms of normalization and these cases would be failing otherwise. Previously, it was assumed that normalization was not required for these test cases, though the cases themselves were not explicitly tailored to this detail (there is a test which tests `normalize=False`). Since the behaviour of `AmpltiudeEmbedding` changed, removing `normalize=False` for each test case could actually be regarded as "adjusting to the new behaviour".

Reply:
I think it's fine - again, we might even drop the argument altogether, because `AmplitudeEmbedding` does not make much sense with an unnormalised state!